packages/vm/src/context.rs

```diff
@@ -139,6 +158,28 @@ pub fn add_iterator<'a, S: Storage, Q: Querier>(
     new_id
 }
 
+pub(crate) fn with_func_from_context<'a, S, Q, Args, Rets, Callback, CallbackData>(
+    ctx: &'a mut Ctx,
```

Comment:
There's no need to explicitly specify `'a` in this case

Reply:
Thanks, done in 85ee836a1bc0ad2c529df7fc16ba96f2d9befa35